### PR TITLE
set up shopping cart connection to backend

### PR DESCRIPTION
**Pull request by jpnoe**
couldn't get a certain part of backend to work, so I had to implement a workaround using a different recipe table entirely

### CODE REVIEW

**Comment by dazen16** [comment]
Looks good. Interesting workaround with a new table

File: backend/model.py [hunk 1]
```diff
@@ -32,37 +32,54 @@ def remove(self):
             self.clear()
             return resp
```
Comment (therodesun) [inline backend/model.py:34]: Perfect!

**Review by therodesun — COMMENTED** [review]
Wonder why we need myrecipe model?

**Comment by Mankler** [comment]
LGTM

**Pull request merged**